etc/docs/build.ts

```diff
@@ -122,48 +60,33 @@ async function main() {
     semverVersion
   };
 
-  const response = await prompt(`
+  await confirm(`
     Generating docs for the following configuration.
 ${JSON.stringify(newVersion, null, 2)}
-    Does this look right? [y/n] `);
+    Does this look right? [y/n] `
+  );
 
-  if (response.trim() !== 'y') {
-    console.error("something went wrong.  Exiting...");
-    process.exit(1);
-  }
-
-  console.error('Installing typedoc...');
-  await exec('npm i --no-save typedoc');
-
-  console.error('Successfully installed typedoc.  Building api docs for current branch...');
+  console.error('Installing dependencies...');
+  await installDependencies();
 
-  await exec('npm run build:typedoc');
+  console.error('Building docs for current branch');
+  await buildDocs();
 
-  console.error('Successfully built api docs for current branch');
+  console.error('Generating new static site...')
 
-  console.error('Generating new doc site...')
   const tomlVersions = parse(await readFile(RELEASES_TOML_FILE, { encoding: 'utf8' })) as unknown as TomlVersionSchema;
   const jsonVersions = JSON.parse(await readFile(RELEASES_JSON_FILE, { encoding: 'utf8' })) as unknown as JsonVersionSchema[];
 
-  const versionAlreadyExists = jsonVersions.some(({version }) => version === semverVersion)
+  const versionAlreadyExists = jsonVersions.some(({ version }) => version === semverVersion)
 
   if (versionAlreadyExists) {
-    const response = await prompt(`Version ${semverVersion} already exists.  Do you want to override the existing docs? [y/n] `);
-    if (response !== 'y') {
-      console.error("something went wrong.  Exiting...");
-      process.exit(1);
-    }
+    await confirm(`Version ${semverVersion} already exists.  Do you want to override the existing docs? [y/n] `);
```

Comment:
I like the idea here, but would it prevent the script from being able to run in the CI? Since we're probably only to run the main generation in CI to ensure docs are kept in sync, maybe we can add a condition to only prompt for the override if the version is not "current".

Also, food for thought, should we give the docs from main some other name, like 
"next" instead of "current" to avoid confusion with the most recently released version?

Reply:
Since Neal kindly pointed out we already have yargs installed, I switched to using yargs and added some extra options. 
 Now it feels like a real command line script 😄 .

- `--yes` will skip any prompts for the user (similar to `npm init -y`)
- `--versionName` will set the display name on the docs page.  i.e., `npm run build:docs -- --versionName next` will display `Next Driver` on the docs 

Personally I prefer this, because then we're not making assumptions in the script and it's more flexible.  What do you think?